roles/swift-account/tasks/main.yml

```diff
@@ -13,6 +13,14 @@
             dest=/etc/swift/account-server.conf owner=swift group=swift
   notify: restart swift-account service
 
+- name: trigger restart on upgrades
```

Comment:
Not all swift account services are restarting.. Same applies for other servers..

Should the following handler be called instead: https://github.com/blueboxgroup/ursula/blob/master/roles/swift-common/handlers/main.yml#L20